<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>366</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>366</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>366</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="digitCount">
         <number>9</number>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="7" column="3">
         <widget class="QPushButton" name="btn9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
           </font>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="btn7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
           </font>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="btn6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
           </font>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="btn3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="btn5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QPushButton" name="btnEqual">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="btn4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="btnMulti">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QPushButton" name="btnPow">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>^</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="btnPlus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="btn2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="2">
         <widget class="QPushButton" name="btn8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
           </font>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="btn1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="btnSubtract">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="btnDivide">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>43</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QPushButton" name="btnRoot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>87</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>27</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>sqrt</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="btn0">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>174</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>44</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QPushButton" name="btnClear">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>165</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>37</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CLEAR</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
